pkgs/tools/misc/clustergit/default.nix

```diff
@@ -0,0 +1,26 @@
+{ stdenv, fetchFromGitHub }:
+
+stdenv.mkDerivation rec {
+  name = "clustergit-${version}";
+  version = "2018-04-18";
+
+  src = fetchFromGitHub {
+    owner = "mnagel";
+    repo = "clustergit";
+    rev = "0d94db95f965c0149eb8423eee991c4d18cf2d59";
+    sha256 = "10v5zq5m1mpzr1xr8y7dkxw654wl9r0hp7n64g4nbjx2ay26yyy4";
+  };
+
+  installPhase = ''
```

Comment:
Here (or during some fixup phase) the script should be properly wrapped with a python with all the required packages available using eg. wrapProgram.